gematria/granite/graph_builder_model_inference.cc

```diff
@@ -412,19 +532,30 @@ GraphBuilderModelInference::FromTfLiteModel(
       *std::move(node_token_list), /* immediate_token = */ *immediate_token,
       /* fp_immediate_token = */ *fp_immediate_token,
       /* address_token = */ *address_token, /* memory_token = */ *memory_token,
+      /* annotation_names */ std::vector<std::string>(),
```

Comment:
It looks like this escaped the split.

Reply:
That needs to be there since `BasicBlockGraphBuilder` already supports annotations, so we just pass in an empty vector to match the argument list - but the behaviour should be the same as before annotations.